defaults/main.yml

```diff
@@ -25,6 +25,12 @@ alpaca_karaf_deploy_dir: "{{ alpaca_karaf_dir }}/deploy"
 alpaca_karaf_user: karaf
 alpaca_local_mvn_path: /opt/maven/repo
 
+alpaca_triplestore_base_url: http://localhost:8080/bigdata
+alpaca_milliner_base_url: http://localhost:8000/milliner/
+alpaca_gemini_base_url: http://localhost:8000/gemini/
+alpaca_houdini_base_url: http://localhost:8000/houdini/convert
+alpaca_homarus_base_url: http://localhost:8000/homarus/convert
```

Comment:
Feels like the name `..._base_url` might be incorrect for these last two.

Reply:
@whikloj endpoing?